src/coreclr/vm/util.cpp

```diff
@@ -1774,24 +1774,14 @@ static BOOL TrustMeIAmSafe(void *pLock)
 
 LockOwner g_lockTrustMeIAmThreadSafe = { NULL, TrustMeIAmSafe };
 
-static DangerousNonHostedSpinLock g_randomLock;
-static CLRRandom g_random;
+static thread_local CLRRandom t_random;
 
 int GetRandomInt(int maxVal)
```

Comment:
There is random `GetRandomInt` in src\coreclr\gc\env\gcenv.base.h that just uses `rand()`. Can it be deleted?